api/src/certification/complementary-certification/application/target-profile-to-complementary-certification-controller.js

```diff
@@ -0,0 +1,26 @@
+import * as complementaryCertificationBadgeSerializer from '../infrastructure/serializers/jsonapi/complementary-certification-badge-serializer.js';
+import { usecases } from '../../shared/domain/usecases/index.js';
+
+const attachBadgesToComplementaryCertification = async function (
```

Comment:
Il faudrait peut-être revoir le nom du fichier et ainsi les autres. Car on a oublié la notion de contextes fonctionnels. On pense encore comme dans `/lib` et non pas comme dans `/src`.

Ici par exemple le fichier pourrait (devrait?) se nommer `attach-target-profile-controller.js`
On a pas besoin du `to-complementary-certification` car on est déjà dans le sous-domaine fonctionnel `complementary-certification` de `certification` (i.e `api/src/certification/complementary-certification/`)

Un nommage plus simple comme `attach-target-profile-controller.js` a donc du sens, car littéralement on peut lire ainsi

> dans mes sources (src) dans le contexte fonctionnel de la certification (certification), et plus précisèment dans le contexte des certifications complémentaires (complementary-certification) je veux attacher un profil cible (attach-target-profile-controller).

Reply:
Je valide, je voulais mettre un commentaire dans ce sens mais entre les 75 rebases j'ai oublié